src/index.html

```diff
@@ -5,8 +5,362 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <span class="image">
+        <img
+          class="image_logo"
+          src="./images/logo.png"
+          alt="Moyo logo"
+        >
+      </span>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#section-1" >Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#section-2">Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#section-3">Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#section-4">Ноутбуки и компьютеры
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#section-5">Гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#section-6">Планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#section-7">Фото
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#section-8">Видео
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+
+    <main class="catalog">
+      <section class="catalog__section">
+        <div class="card"
+          data-qa="card"
+        >
```

Comment:
better leave one tag - `section or div`. And use mix to write classes.
More detail in video.
[video, timing 4:15](https://mate.academy/learn/css-advanced/bem#/video/911)

Add changes to styles according to changes in html.
Add the same changes for all cards.
```suggestion
    <main class="catalog">
        <div class="card cataloge__card"
          data-qa="card"
        >
```